malaria-ios/ViewControllers/RapidFireGame.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
+
+class RapidFireGame: Game {
+  
+  var entries: [RapidFireGameEntry] = []
+  
+  init() {
+    let entry1 = RapidFireGameEntry(question: "Question 1 (answer 2)", answers: ["Answer 1", "Answer 2", "Answer 3"], correctAnswer: 1)
+    let entry2 = RapidFireGameEntry(question: "Question 2 (answer 1)", answers: ["Answer 1", "Answer 2", "Answer 3"], correctAnswer: 0)
+    let entry3 = RapidFireGameEntry(question: "Question 3 (answer 3)", answers: ["Answer 1", "Answer 2", "Answer 3"], correctAnswer: 2)
+    
+    entries.appendContentsOf([entry1, entry2, entry3])
```

Comment:
Is it possible to import this questions from an JSON/Yaml/Xml/etc file? I think we have already some methods to import from JSON file (it would be a one step ahead if we later receive this from the server) ) ;)